Polygon_repair_2/examples/Polygon_repair_2/clipart.cpp

```diff
@@ -0,0 +1,151 @@
+#include <iostream>
```

Comment:
This should go in test IMO

Reply:
What about the clipart data for this?

Reply:
Since we cannot include it, maybe the best is to move the .cpp in benchmark, and add a comment about where to get the data.